src/components/BlockThreeColumnList.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import get from 'utils/get';
+
+import { Markdown } from 'components/base';
+
+const BlockThreeColumnList = props => {
+  const fields = get(props, 'block.fields');
+  const header = get(fields, 'header', '');
+  const columnOne = get(fields, 'columnOne', '');
+  const columnTwo = get(fields, 'columnTwo', '');
+  const columnThree = get(fields, 'columnThree', '');
+  const marginBottom = get(fields, 'marginBottom', 0);
+  const marginTop = get(fields, 'marginTop', 0);
+
+  return (
+    <div
+      style={{
+        marginBottom: `${marginBottom}rem`,
+        marginTop: `${marginTop}rem`
+      }}
+      className="BlockThreeColumnList flex flex-col col-8 md:col-6 mxauto px1 md:px0 pt3 md:pt0 pb2 md:pb7"
+    >
+      {header && (
+        <h1 className="BlockThreeColumnList__header small col-8 md:col-4 pb2">{header}</h1>
+      )}
+        <div className="flex flex-col md:flex-row justify-between">
+          <div className="pb2 md:pb0 md:col-2">
+            <span className="small">1.</span>
```

Comment:
This number can come from CMS? so that we can use it with or without it?

Reply:
Brendon said they will always have these numbers and always be 3 columns! Still remove? :) 
Also sorry Feature Peek isn't working currently (they have a caching issue), it will be up soon and you will be able to go to `/style-guide`!

Reply:
👍 agree w/ sohee - more flexible is more betterer